sale_timesheet_existing_project/models/sale_order.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Tecnativa -  Pedro M. Baeza
```

Comment:
Why this extra space?

Reply:
Removed (it was a trick to force pip custom library creation)